pkg/storaged/dialog.jsx

```diff
@@ -1161,13 +1163,15 @@
 }
 
 export const StopProcessesMessage = ({ mount_point, users }) => {
+    const mock = useMock()?.busy;
+
     const process_rows = users.filter(u => u.pid).map(u => {
         return {
             columns: [
-                u.pid,
+                mock?.pid ?? u.pid,
                 { title: u.cmd.substr(0, 100), props: { modifier: "breakWord" } },
                 u.user || "-",
-                { title: format_delay(-u.since * 1000), props: { modifier: "nowrap" } }
+                { title: mock?.since ?? format_delay(-u.since * 1000), props: { modifier: "nowrap" } }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20230302-122128-2c63e908-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)